docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -603,251 +619,262 @@ Specific observer outputs and their stratifications may vary by model run as nee
 5.0 Model run requests
 ++++++++++++++++++++++
 
-.. list-table:: Model run plan as of October 4, 2024
-  :widths: 3 3 3 3 3 3 5 15
+.. list-table:: Default simulation specifications
+  :header-rows: 1
+
+  * - Parameter
+    - Value
+    - Note
+  * - Location(s)
+    - * Ethiopia (ID: 179)
+      * Nigeria (ID: 214)
+      * Pakistan (ID: 165)
+    -
+  * - Number of draws
+    - 10
+    - Based on calculations from the `Nutrition Optimization project <https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.html#production-run-specifications>`_: production run number divided in half for default V&V runs
+  * - Population size per draw
+    - 100,000
+    - Based on calculations from the `Nutrition Optimization project <https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.html#production-run-specifications>`_: production run number divided in half for default V&V runs
+  * - Randomness key columns
+    - ['entrance_time','age']
```

Comment:
do we need to make sure that the age value changes between timesteps because if it does not we will not refresh our randomness key???

Reply:
Rajan says we get a new randomness key on every timestep even if age does not update... but is 99% sure and will check